platform-coop-toolkit/coop-map-directory-index
# feat: add my profiles flow (resolves #28)

## Description

Replaces profiles in menu with "My Profiles" page. Allows profiles to be created and deleted.

## Steps to test

1. Check out this pull request.
2. Navigate to `/my-profiles/`.
3. If no profiles are present, create both an individual and organizational profile, then return to the page.
4. Confirm that both individual and organizational profiles can be deleted with confirmation prompt.
5. Log out.
6. Confirm that page is not shown.

## Additional information

This page replaces both the "My profiles" dropdown and the "Manage profiles" page seen in XD.

## Related issues

- Resolves #28.
- Creating organization profiles requires #84.
